types/vue.d.ts

```diff
@@ -80,10 +80,28 @@ declare namespace NuxtVueI18n {
  * Extends types in vue
  */
 declare module "vue/types/vue" {
+  interface NuxtI18nSeo {
+    htmlAttrs: {
```

Comment:
Should htmlAttrs, link and meta be marked as optional? Logic of seo-head.js doesn't guarantee them to be set.

Reply:
@rchl i messed up syntax and declared them wrong as a array of 1 element :) Changed.
htmlAttrs, link and meta will always be returned as i see from a code. But they could be empty arrays. And lang in htmlAttrs could be optional. I changed that.

Reply:
actually i missed first return, so yes they should be optional